### DISCUSSION TITLE
Azure - Can it calculate emissions for MCA type account ?

### DISCUSSION THREAD

**Discussion by nshelia** [cloud-carbon-footprint/cloud-carbon-footprint#1249] [Q&A]
https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields

I see that exporting billings reports for MCA type accounts produces `CostInBillingCurrency` field in CSV but as I see repository only uses `CostInUSD` or  `Cost` calculations. Is this correct ? Does this mean I can't use it for MCA type Azure accounts ?

**Comment by camcash17** [comment]
Hi @nshelia, thanks for raising this. Currently, we have this [open issue ](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/1224) that details and links to a lot of context on this subject. We are actively looking for support to help implemented feature enhancements for CCF. This may entail handling the `CostInBIllingCurrency` field.

Please let us know if you would have any interest in helping update the handling of Azure billing for CCF!
